lldb/include/lldb/Symbol/UnwindPlan.h

```diff
@@ -446,18 +446,15 @@ class UnwindPlan {
         m_plan_is_for_signal_trap(rhs.m_plan_is_for_signal_trap),
         m_lsda_address(rhs.m_lsda_address),
         m_personality_func_addr(rhs.m_personality_func_addr) {
-    m_row_list.reserve(rhs.m_row_list.size());
-    for (const RowSP &row_sp : rhs.m_row_list)
-      m_row_list.emplace_back(new Row(*row_sp));
+    for (const auto &[offset, row_sp] : rhs.m_rows)
+      m_rows.emplace(offset, std::make_shared<Row>(*row_sp));
   }
 
   ~UnwindPlan() = default;
 
   void Dump(Stream &s, Thread *thread, lldb::addr_t base_addr) const;
 
-  void AppendRow(const RowSP &row_sp);
-
-  void InsertRow(const RowSP &row_sp, bool replace_existing = false);
+  void InsertRow(const RowSP &row_sp, bool replace_existing = true);
```

Comment:
Changed the default because that's how AppendRow worked (InsertRow was only called from test code until now)